demo/chat-room/style.css

```diff
@@ -1,151 +1,111 @@
-.main{
+.main {
   margin: 20px;
 }
 
-.myblock{
-  margin:20px;
+.myblock {
+  margin: 20px;
 }
 
-.conversation-wrap
-{
+.conversation-wrap {
     box-shadow: -2px 0 3px #ddd;
     padding:0;
-    max-height: 400px;
+    /* max-height: 400px; */
     overflow: auto;
+    background: #f9f7f7;
 }
-.conversation
-{
-    padding:5px;
-    border-bottom:1px solid #ddd;
-    margin:0;
 
+small {
+    color: #7fb6ff;
 }
 
-.message-wrap
-{
-    box-shadow: 0 0 3px #ddd;
-    padding:0;
+.conversation {
+    padding: 5px;
+    border-bottom: 1px solid #ddd;
+    margin: 0;
+}
 
+.message-wrap {
+    box-shadow: 0 0 3px #ddd;
+    padding: 0;
 }
-.msg
-{
+
+.msg {
     padding:5px;
-    /*border-bottom:1px solid #ddd;*/
     margin:0;
 }
-.msg-wrap
-{
+
+.msg-wrap {
     padding:10px;
     height: 60%;
     overflow: auto;
-
+    background: #f9f7f7;
 }
 
-.time
-{
+.time {
     color:#bfbfbf;
 }
 
-.send-wrap
-{
+.send-wrap {
     border-top: 1px solid #eee;
     border-bottom: 1px solid #eee;
     padding:10px;
-    /*background: #f8f8f8;*/
-    }
-
-    .send-message
-    {
-        resize: none;
-    }
-
-    .highlight
-    {
-        background-color: #f7f7f9;
-        border: 1px solid #e1e1e8;
-    }
-
-    .send-message-btn
-    {
-        border-top-left-radius: 0;
-        border-top-right-radius: 0;
-        border-bottom-left-radius: 0;
-        border-bottom-right-radius: 0;
-    }
-    .btn-panel
-    {
-        background: #f7f7f9;
-    }
-
-    .btn-panel .btn
-    {
-        color:#696969;
-        transition: 0.2s all ease-in-out;
-    }
-
-    .btn-panel .btn:hover
-    {
-        color:#666;
-        background: #f8f8f8;
-    }
-    .btn-panel .btn:active
-    {
-        background: #f8f8f8;
-        box-shadow: 0 0 1px #ddd;
-    }
-
-    .btn-panel-conversation .btn,.btn-panel-msg .btn
-    {
-
-        background: #f8f8f8;
-    }
-    .btn-panel-conversation .btn:first-child
-    {
-        border-right: 1px solid #ddd;
-    }
-
-    .msg-wrap .media-heading
-    {
-        color:#003bb3;
-        font-weight: 700;
-    }
-
-
-    .msg-date
-    {
-        background: none;
-        text-align: center;
-        color:#aaa;
-        border:none;
-        box-shadow: none;
-        border-bottom: 1px solid #ddd;
-    }
-
-
-    body::-webkit-scrollbar {
-        width: 12px;
-    }
- 
-    
-    /* Let's get this party started */
+    background: #f9f7f7;
+}
+
+.send-message {
+    resize: none;
+}
+
+.highlight {
+    background-color: #f7f7f9;
+    border: 1px solid #e1e1e8;
+}
+
+.send-message-btn {
+    border-top-left-radius: 0;
```

Comment:
border-radius: 0 could work here

Reply:
It does not because we only need the upper part